src/DynamoCoreWpf/ViewModels/Core/ConnectorPinViewModel.cs

```diff
@@ -51,7 +51,7 @@ public virtual void OnRequestRemove(Object sender, EventArgs e)
         #region Properties
 
         [JsonIgnore]
-        private readonly WorkspaceViewModel WorkspaceViewModel;
+        public readonly WorkspaceViewModel WorkspaceViewModel;
```

Comment:
whats the reason for making this public now?

Reply:
Yeah I think it is a vestige of something I was trying. Made private again.